src/main/java/Duke.java

```diff
@@ -1,10 +1,240 @@
+import java.util.Random;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class Duke {
+    private static File save;
+
+    public static String addTaskSalutation(String item) {
+        Random rand = new Random();
+        String out = "";
+        switch(rand.nextInt(3)) {
+        case 0:
+            out = "Right away sir. I have added " + item + " to your list of tasks.\n";
+            break;
+        case 1:
+            out = "Understood sir. " + item + " has been added to your list of tasks.\n";
+            break;
+        case 2:
+            out = "Sir, I can confirm that " + item + " has been added to your list of tasks.\n";
+            break;
+        default:
+            break;
+        }
+        return out;
+    }
+    public static void line() {
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    public static void showList(Tasks user) {
+        for (int i = 0; i < user.listLength; i++) {
+            System.out.println("\t" + (i + 1) + ". " + user.list[i] + "\n");
+        }
+    }
+    public static void addTodo(String input, Tasks user) {
```

Comment:
You may wish to consider renaming the `user` parameter. It is strange that the parameter is `user` of type `Tasks`, when a task usually refers to object/item rather than a person.